Gems/Multiplayer/Code/Source/NetworkEntity/NetworkEntityManager.cpp

```diff
@@ -425,47 +426,55 @@ namespace Multiplayer
 
             // Can't use NetworkEntityTracker to do the lookup since the entity has not activated yet
             NetBindComponent* netBindComponent = clone->FindComponent<NetBindComponent>();
-            if (netBindComponent != nullptr)
+            if (!netBindComponent)
+            {
+                delete clone;
+                continue;
+            }
```

Comment:
Is there a reason we check for NetBindComponent on the clone vs the original? Seems like you could check the original and early out without the Clone step.

Reply:
Good call! Fixed 👍